internal/service/verify/email.go

```diff
@@ -0,0 +1,72 @@
+package verifysvc
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/resend/resend-go/v2"
+)
+
+type SendParams struct {
+	Email  string
+	UserId string
+}
+
+func (s *Service) Send(ctx context.Context, sendparams SendParams) error {
+	apiKey := s.cfg.ResendApiKey
+	client := resend.NewClient(apiKey)
+
+	verify, err := s.db.User.Verify.Create(ctx, sendparams.UserId)
+	if err != nil {
+		return err
```

Comment:
```suggestion
		return fmt.Errorf("failed to create verification: %w", err)
```
Write appropriate error messages